### ISSUE TITLE
Relaxing with Alonso causes heart to explode

### ISSUE THREAD

**Issue by Brinabavd** [CleverRaven/Cataclysm-DDA#45211]
"Relaxing with Alonso" kills player

"Relaxing with Alonso" kills player by setting stim value to 1000

# Steps To Reproduce

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1909
- Game Version: 0.E-7193-g1ae4a80 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    C.R.I.T Expansion Mod [crt_expansion],
    Blaze Industries [blazeindustries],
    Aftershock [aftershock],
    Alternative Map Key [alt_map_key],
    Crazy Cataclysm [crazy_cataclysm],
    Mythical Martial Arts [MMA],
    Disable NPC Needs [no_npc_food],
    Fuji's Military Profession Pack [fuji_mpp],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills],
    Graphical Overmap [Graphical_Overmap]
]

# Additional context

**Comment by Aivean** [comment]
On a serious note, probably a bug:
```json
  {
    "type": "effect_type",
    "id": "al_relaxed",
    "name": [ "Relaxed with Alonso" ],
    "apply_message": "You spent some pleasant time relaxing with someone.",
    "miss_messages": [ [ "You feel relaxed.", 1 ] ],
    "rating": "good",
    "base_mods": { "stim_amount": [ 50 ], "stim_min": [ 50 ] }
  },
```

**Comment by int-ua** [comment]
Removing the `Bug` label until a fresh confirmation. Please add steps to reproduce.

**Comment by Saicchi** [comment]
Can confirm on 0.E-9884-g2f0f0585b4 [64-bit]
1. Complete Alonso pants quest
2. Have a condom in inventory and pick the options that give the `"Relaxed with Alonso"` effect.
3. Wait a turn and see your stimulants shoot up
4. After a few seconds the player reaches 1000 stimulants and stimulants overdose.

**Comment by actual-nh** [comment]
Perhaps add `"stim_max": [ 150 ]` to the `base_mods` field?

**Comment by Saicchi** [comment]
Did you mean `stim_max_val` ? Unless I'm mistaken about how effects work. Even so 150 seems quite high.

**Comment by actual-nh** [comment]
> Did you mean `stim_max_val` ? Unless I'm mistaken about how effects work. Even so 150 seems quite high.

150 was 3x the base one - see #47831.

> And yes, I also agree that a morale effect would be better in this case.

As far as I can tell, not currently possible for effects - I have suggested removal of a couple of emotional negative effects.

**Comment by casswedson** [comment]
Fixed by #47894
